test/wolt_modal_sheet_test.dart

```diff
@@ -3,57 +3,68 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:wolt_modal_sheet/wolt_modal_sheet.dart';
 
 void main() {
+  Widget buildSheetWithShow({
```

Comment:
Why does this have to be a method? Can't we just create a `WidgetTestApp` and pump it?

Reply:
It simply to return a widget, we don't need context or state if you're "WidgetTestApp" to be a widget class.